Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,17 @@
+function calibrateFrequency(changes) {
+    let currentFrequency = 0;
+
+    for (const change of changes) {
+        currentFrequency += parseInt(change, 10);
+    }
+
+    return currentFrequency;
+}
+
+const frequencyChanges = [ "+1",  "-1", "+2", "-2", "+3", "-3"]
```

Comment:
I think this exercise expect you to also write JS code to read the numbers from the file `input.txt` directly.

You have done the "calculating the sum of the frequencies" part. You just need to add code to read the data from the given data file and store the frequencies in an array.